************* Module Limited_Access_Factor_Models
python-scripts\Limited_Access_Factor_Models.py:1:0: C0114: Missing module docstring (missing-module-docstring)
python-scripts\Limited_Access_Factor_Models.py:1:0: C0103: Module name "Limited_Access_Factor_Models" doesn't conform to snake_case naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:40:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:42:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:73:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:74:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:74:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:76:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:91:0: C0413: Import "import matplotlib.pyplot as plt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:92:0: W0404: Reimport 'datetime' (imported line 73) (reimported)
python-scripts\Limited_Access_Factor_Models.py:92:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:93:0: C0413: Import "from gs_quant.timeseries import beta" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:94:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:94:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:96:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:119:0: W0404: Reimport 'datetime' (imported line 73) (reimported)
python-scripts\Limited_Access_Factor_Models.py:119:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:120:0: C0413: Import "import matplotlib.ticker as mtick" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:121:0: W0404: Reimport 'matplotlib.pyplot' (imported line 91) (reimported)
python-scripts\Limited_Access_Factor_Models.py:121:0: C0413: Import "import matplotlib.pyplot as plt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:122:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:122:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:124:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:169:0: W0404: Reimport 'datetime' (imported line 73) (reimported)
python-scripts\Limited_Access_Factor_Models.py:169:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:170:0: C0413: Import "import pandas as pd" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:171:0: C0413: Import "from gs_quant.models.risk_model import RiskModelUniverseIdentifierRequest as Identifier, DataAssetsRequest" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:172:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:172:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:175:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:182:0: C0103: Constant name "max_rows" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:184:0: W0104: Statement seems to have no effect (pointless-statement)
python-scripts\Limited_Access_Factor_Models.py:193:0: W0404: Reimport 'datetime' (imported line 73) (reimported)
python-scripts\Limited_Access_Factor_Models.py:193:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:194:0: W0404: Reimport 'matplotlib.pyplot' (imported line 91) (reimported)
python-scripts\Limited_Access_Factor_Models.py:194:0: C0413: Import "import matplotlib.pyplot as plt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:195:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:195:0: W0404: Reimport 'DataAssetsRequest' (imported line 171) (reimported)
python-scripts\Limited_Access_Factor_Models.py:195:0: W0404: Reimport 'RiskModelUniverseIdentifierRequest' (imported line 171) (reimported)
python-scripts\Limited_Access_Factor_Models.py:195:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel, DataAssetsRequest, RiskModelUniverseIdentifierRequest as Identifier" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:197:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:200:0: C0103: Constant name "asset_bbid" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:222:0: C0413: Import "import seaborn as sns" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:223:0: W0404: Reimport 'datetime' (imported line 73) (reimported)
python-scripts\Limited_Access_Factor_Models.py:223:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:224:0: W0404: Reimport 'matplotlib.pyplot' (imported line 91) (reimported)
python-scripts\Limited_Access_Factor_Models.py:224:0: C0413: Import "import matplotlib.pyplot as plt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:226:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:226:0: W0404: Reimport 'DataAssetsRequest' (imported line 171) (reimported)
python-scripts\Limited_Access_Factor_Models.py:226:0: W0404: Reimport 'RiskModelUniverseIdentifierRequest' (imported line 171) (reimported)
python-scripts\Limited_Access_Factor_Models.py:226:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel, DataAssetsRequest, RiskModelUniverseIdentifierRequest as Identifier" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:228:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:231:0: C0103: Constant name "asset_bbid" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:250:0: W0404: Reimport 'datetime' (imported line 73) (reimported)
python-scripts\Limited_Access_Factor_Models.py:250:0: C0413: Import "import datetime as dt" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:251:0: W0404: Reimport 'FactorRiskModel' (imported line 40) (reimported)
python-scripts\Limited_Access_Factor_Models.py:251:0: W0404: Reimport 'DataAssetsRequest' (imported line 171) (reimported)
python-scripts\Limited_Access_Factor_Models.py:251:0: W0404: Reimport 'RiskModelUniverseIdentifierRequest' (imported line 171) (reimported)
python-scripts\Limited_Access_Factor_Models.py:251:0: C0413: Import "from gs_quant.models.risk_model import FactorRiskModel, Measure, DataAssetsRequest, RiskModelUniverseIdentifierRequest as Identifier" should be placed at the top of the module (wrong-import-position)
python-scripts\Limited_Access_Factor_Models.py:253:0: C0103: Constant name "model_id" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:260:0: C0103: Constant name "asset_bbid" doesn't conform to UPPER_CASE naming style (invalid-name)
python-scripts\Limited_Access_Factor_Models.py:73:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:92:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:119:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:169:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:193:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:223:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:250:0: C0411: standard import "import datetime as dt" should be placed before "from gs_quant.session import Environment, GsSession" (wrong-import-order)
python-scripts\Limited_Access_Factor_Models.py:93:0: C0412: Imports from package gs_quant are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:120:0: C0412: Imports from package matplotlib are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:122:0: C0412: Imports from package gs_quant are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:171:0: C0412: Imports from package gs_quant are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:194:0: C0412: Imports from package matplotlib are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:195:0: C0412: Imports from package gs_quant are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:224:0: C0412: Imports from package matplotlib are not grouped (ungrouped-imports)
python-scripts\Limited_Access_Factor_Models.py:226:0: C0412: Imports from package gs_quant are not grouped (ungrouped-imports)

------------------------------------------------------------------
Your code has been rated at 3.39/10 (previous run: 3.39/10, +0.00)

